README.md

```diff
@@ -108,7 +114,7 @@ const expenses = sum(budget, ...form.elements.expenses);
 
 These are then caught and surfaced to the user in a human-friendly message (such as via a "toast").
 
-### TDLR
+### TD;LR
```

Comment:
It can be written either way, but including the semicolon I believe makes the url hash ugly and difficult to guess.